l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -584,7 +586,8 @@ def setDettaglioLinee(self, invoice, body):
             DettaglioLinea = DettaglioLineeType(
                 NumeroLinea=str(line_no),
                 Descrizione=line.name,
-                PrezzoUnitario='%.2f' % prezzo_unitario,
+                PrezzoUnitario=eval(
+                    "'%.' + str(price_precision) + 'f'") % prezzo_unitario,
                 Quantita='%.2f' % line.quantity,
```

Comment:
Could you please do the same with `quantity`? 
You could use `self.env['decimal.precision'].precision_get('Product Unit of Measure')`

Reply:
Done